src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,44 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String brand;
+    private String model;
+    private int horsePower;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, String brand, String model, int horsePower) {
+        super();
+        this.brand = brand;
+        this.model = model;
+        this.horsePower = horsePower;
+    }
```

Comment:
Right now you take 5 parameters, but you use only 3, because using empty super, `super()` just calls Machine's empty constructor, you have to pass params inside to call a proper constructor in the Machine class.
```suggestion
    public Bulldozer(String name, String color, String brand, String model, int horsePower) {
        super(name, color);
        this.brand = brand;
        this.model = model;
        this.horsePower = horsePower;
    }
```

Reply:
Same for Truck and Excavator